packages/agents-hosting/src/activityHandler.ts

```diff
@@ -30,12 +30,33 @@ const logger = debug('agents:activity-handler')
 /**
  * Handles incoming activities from channels and dispatches them to the appropriate handlers.
  *
- * The ActivityHandler provides a foundation for handling different types of activities
- * in an agent application. It processes various activity types such as messages, conversation updates,
- * message reactions, etc., and routes them to the appropriate handler methods.
+ * @remarks
```

Comment:
we might want to call out that we recommend to use the AgentApplication class instead, this is provided only for back compat purposes.